specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -1220,6 +1220,11 @@
           "description": "Gets or sets the properties of the Dsc Compilation job."
         }
       },
+      "allOf": [
```

Comment:
If adding proxyResource, please remove "name" and "id" (duplicate)

Reply:
Resolved.